docs/src/basicproperties.md

```diff
@@ -8,7 +8,7 @@ The following is an overview of functions for accessing graph properties. For fu
 - `vertices`: Iterable object of all graph vertices.
 - `edges`: Iterable object of all graph edges.
 - `has_vertex`: Checks for whether graph includes a vertex.
-- `has_edge`: Checks for whether graph includes an edge.
+- `has_edge`: Checks for whether graph includes an edge from a given source to a given desination.
```

Comment:
(Note typo in `destination`.) Would suggest: 

```
Checks for whether graph includes an edge from a given source to a given destination.
In most cases, `has_edge` can also check whether a graph includes an edge `e` that is
equal to another edge in the graph (equality being defined by the specific graph type).
```

Reply:
why "in most cases", if you don't define it, it has a default implementation so it will check it

Reply:
so "`has_edge` can also check..." works fine

Reply:
I think we need more explanatory text - either here, or somewhere else, along these lines:

> `has_edge(g, i, j)` will return `true` if there is an edge between vertices `i` and `j` in graph `g`. This version of `has_edge` ignores any other metadata (such as weight) associated with the edge.
> `has_edge(g, e)` will return `true` if there is an edge in `g` that satisfies `e == f` for any `f ∈ edges(g)`. This is a strict equality test that may not limit itself for testing that just there exists any edge between the source and destination.
> Note: to use the `has_edge(g, e)` method safely, it is important to understand the conditions under which edges are equal to each other. This will necessarily vary among edge types and defaults to `src(e) == src(f) && dst(e) == dst(f)`.